src/pascy/l2.py

```diff
@@ -13,18 +13,77 @@ class ArpLayer(Layer):
     @staticmethod
     def fields_info():
         # TODO: Implement this :)
-        pass
+        return [ UnsignedShort("hardware_type", 1), UnsignedShort("protocol_type", 0x800),
+                UnsignedByte("hardware_size", 6), UnsignedByte("protocol_size", 4),
+                UnsignedShort("opcode", ArpLayer.OP_IS_AT), 
+                MacAddress("mac_src"), IPAddress("ip_src"),
+                MacAddress("mac_dst", MAC_BROADCAST), 
+                IPAddress("ip_dst", IP_BROADCAST)]
+
+
+class IcmpLayer(Layer):
+    PING_REQUEST = 8
+    PING_REPLY = 0
+
+    NAME = "ICMP"
+
+    @staticmethod
+    def fields_info():
+        return [UnsignedByte('type', IcmpLayer.PING_REPLY), UnsignedByte('code', 0), UnsignedShort('checksum', 0),
```

Comment:
Break down to one item per line.
It makes it much more readable, and easier for git diffs